junit-jupiter-api/src/main/java/org/junit/jupiter/api/AssertionUtils.java

```diff
@@ -118,6 +127,10 @@ static boolean floatsAreEqual(float value1, float value2, float delta) {
 		return floatsAreEqual(value1, value2) || Math.abs(value1 - value2) <= delta;
 	}
 
+	static boolean floatsAreEqual(float value1, float value2) {
+		return Float.floatToIntBits(value1) == Float.floatToIntBits(value2);
+	}
+
```

Comment:
I moved it because, before I added "-Xep:UngroupedOverloads:OFF" to the `build.gradle` file, error-prone reported that methods like this one were not 'grouped' together with methods of the same name, potentially causing confusion for future maintainers.

I ultimately added "-Xep:UngroupedOverloads:OFF" to the `build.gradle` because the corresponding rule produced a lot of churn for little gain, but as I was reverting the changes I made initially to satisfy the rule, I thought that moving all the `floatsAreEqual()` methods together made a great deal of sense, so I decided to keep it.

If you want me to move it back to where it was, or to put this change in a new PR, please let me know!